controllers/front/CartController.php

```diff
@@ -435,8 +446,8 @@ protected function processChangeProductInCart()
         // Check product quantity availability
         if ('update' !== $mode && $this->shouldAvailabilityErrorBeRaised($product, $qty_to_check)) {
             $this->{$ErrorKey}[] = $this->trans(
-                'The product is no longer available in this quantity.',
-                [],
+                'The product %product% is no longer available in this quantity.',
+                ['%product%' => $product->name . (isset($productAttributes) ? ' ' . $productAttributes : '')],
```

Comment:
Will the attributes be translatable?

Reply:
Yes, they are retrieved from this function https://github.com/PrestaShop/PrestaShop/blob/2c31258e7f9bdb1679cae50b05ffabca8d9fafe1/classes/Product.php#L7247 which consider context language to get the string